sdcm/utils/common.py

```diff
@@ -1951,3 +1951,26 @@ def ec2_ami_get_root_device_name(image_id, region):
             return image.root_device_name
     except (TypeError, ClientError):
         raise AssertionError(f"Image '{image_id}' details not found in '{region}'")
+
+
+def get_snapshot_content(snapshot_name, listsnapshots_output) -> list():
```

Comment:
add type hints to the function arguments

Reply:
`get_snapshot_content` -> `parse_nodetool_listsnapshots`

Reply:
the function should return dict with the details for each snapshot:

```python
SnapshotDetails = namedtuple("SnapshotDetails", ["keyspace_name", "table_name"])
...
{
    "1599414845162": SnapshotDetails(keyspace_name='system_schema', table_name='keyspaces'),
}

Reply:
Done